kubetest/e2e/runner.go

```diff
@@ -0,0 +1,139 @@
+package e2e
+
+import (
+	"fmt"
+	"log"
+	"os/exec"
+	"strings"
+
+	"k8s.io/test-infra/kubetest/process"
+)
+
+type Tester struct {
+	// Required
+	Kubeconfig string
+	Provider   string
+
+	// Populated with reasonable default values
+	GinkgoPath string
+
+	GinkgoParallel int
+
+	// Other options defined in hack/ginkgo.sh
+	KubeMasterURL         string
+	FlakeAttempts         int
+	GCEProject            string
+	GCEZone               string
+	GCERegion             string
+	GCEMultizone          bool
+	GKECluster            string
+	KubeMaster            string
+	ClusterID             string
+	CloudConfig           string
+	KubeRoot              string
+	NodeInstanceGroup     string
+	KubeGCEInstancePrefix string
+	Network               string
+	NodeTag               string
+	MasterTag             string
+	ClusterMonitoringMode string
+	KubeContainerRuntime  string
+	MasterOSDistribution  string
+	NodeOSDistribution    string
+	NumNodes              int
+	ReportDir             string
+	ReportPrefix          string
+}
+
+func NewTester() *Tester {
+	t := &Tester{
+		FlakeAttempts: 1,
+		GinkgoPath:    "ginkgo",
```

Comment:
This changes if we are in bazel build or go build. Can we just do a bazel run???